website/docs/tutorial/set-up-data-platform.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Set up your data platform
+id: set-up-data-platform
+description: "Set up your environment to get started using dbt."
+---
+
+Now that you have some context for this tutorial, let's get your environment set up. Check out the table of contents to the left to set up your environment on BigQuery, Snowflake, or Redshift. These guides are helpful for anyone with an existing data platform or starting from scratch.
```

Comment:
praise: Love that you broke this up into it's own page to give context rather than at the bottom of the previous page of the tutorial.  It chunks it and makes it really clear that this is a decision the reader need to make.